### ISSUE TITLE
Group Project: Wireframes Homework

### ISSUE THREAD

**Issue by SoItGoes33** [SoItGoes33/group-proj#1]
Hey Soumia and Carol -

Excited to be working on this project with both of you! I was wondering how both of you wanted to handle our group homework for next week regarding creating the wireframes.  I won't be able to attend the informal code brunch on Sunday (nor really meet on Sunday at all because I have to be in Monterey all day as I'm trying to get my scuba diving certification).  Would getting together on Saturday work for you guys? Or do you just want to coordinate online? Please let me know what you'd like to do.

Also, here is the link to Tracy Chou's github account where she's been gathering all the info related to the diversity stats in tech: https://github.com/triketora

**Comment by ghost** [comment]
Hi guys,

Super stoked to meetup and coordinate this.  Saturday works for me and I'd
also love to organize a little beforehand online.  Feel free to email me at
soumiafares@gmail.com (I don't have your email addresses) so we can
coordinate.

Thanks!!

On Thu, Aug 27, 2015 at 10:42 AM, Sullivan Fisker notifications@github.com
wrote:

> Hey Soumia and Carol -
> 
> Excited to be working on this project with both of you! I was wondering
> how both of you wanted to handle our group homework for next week regarding
> creating the wireframes. I won't be able to attend the informal code brunch
> on Sunday (nor really meet on Sunday at all because I have to be in
> Monterey all day as I'm trying to get my scuba diving certification). Would
> getting together on Saturday work for you guys? Or do you just want to
> coordinate online? Please let me know what you'd like to do.
> 
> Also, here is the link to Tracy Chou's github account where she's been
> gathering all the info related to the diversity stats in tech:
> https://github.com/triketora
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/SoItGoes33/group-proj/issues/1.

## 
- Soumia N. Fares

**Comment by carolc** [comment]
Hi all! Yay!

I unfortunately will not be around Saturday, happy to email back and forth.
If you two want to meet, I can send in my thoughts beforehand. I'm also
fairly experienced with wireframing so can definitely help a lot with that!
My email is carollyc@gmail.com if you want to start an email thread.

See you soon!
Carol

**Comment by ghost** [comment]
Stoked to hear this Carol!  I started a little thought doc just not (I'm at
work) which you guys can view/edit/comment here
https://docs.google.com/document/d/1U6yT5c0iM_xEc6zlNQj1sJy4BxSyEkcdMtEHRpXNiR0/edit?usp=sharing

We can bounce ideas on it back and forth or in an email thread.  Excited to
hear your thoughts guys!! :)

On Thu, Aug 27, 2015 at 1:59 PM, Carol C notifications@github.com wrote:

> Hi all! Yay!
> 
> I unfortunately will not be around Saturday, happy to email back and forth.
> If you two want to meet, I can send in my thoughts beforehand. I'm also
> fairly experienced with wireframing so can definitely help a lot with that!
> My email is carollyc@gmail.com if you want to start an email thread.
> 
> See you soon!
> Carol
> 
> On Thu, Aug 27, 2015 at 1:25 PM, Soumia Fares notifications@github.com
> wrote:
> 
> > Hi guys,
> > 
> > Super stoked to meetup and coordinate this. Saturday works for me and I'd
> > also love to organize a little beforehand online. Feel free to email me
> > at
> > soumiafares@gmail.com (I don't have your email addresses) so we can
> > coordinate.
> > 
> > Thanks!!
> > 
> > On Thu, Aug 27, 2015 at 10:42 AM, Sullivan Fisker <
> > notifications@github.com>
> > wrote:
> > 
> > > Hey Soumia and Carol -
> > > 
> > > Excited to be working on this project with both of you! I was wondering
> > > how both of you wanted to handle our group homework for next week
> > > regarding
> > > creating the wireframes. I won't be able to attend the informal code
> > > brunch
> > > on Sunday (nor really meet on Sunday at all because I have to be in
> > > Monterey all day as I'm trying to get my scuba diving certification).
> > > Would
> > > getting together on Saturday work for you guys? Or do you just want to
> > > coordinate online? Please let me know what you'd like to do.
> > > 
> > > Also, here is the link to Tracy Chou's github account where she's been
> > > gathering all the info related to the diversity stats in tech:
> > > https://github.com/triketora
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > https://github.com/SoItGoes33/group-proj/issues/1.
> > 
> > ## 
> > - Soumia N. Fares
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/SoItGoes33/group-proj/issues/1#issuecomment-135544450>
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/SoItGoes33/group-proj/issues/1#issuecomment-135553626
> .

## 
- Soumia N. Fares

**Comment by carolc** [comment]
Do you guys want to meet with the instructors before or after class on
Monday to talk about scope??

**Comment by ghost** [comment]
I think that's a great idea but I work and wouldn't be able to meet before class. I can stick around after for sure.

**Comment by SoItGoes33** [comment]
Yeah, I probably won't be able to meet before class either since I don't
really get back to the city from the South Bay until 5:40.  I could stay
after class also.
